rsts/getting_started/index.rst

```diff
@@ -207,7 +229,7 @@ Then run the same Workflow on the Flyte cluster:
 
 |
 
-.. dropdown:: :fa:`info-circle` What does the ``--remote`` flag do?
+.. dropdown:: :fa:`info-circle` The role of ``--remote`` and ``--image`` flags
```

Comment:
Let's have a separate dropdown, smriti. Including --image here doesn't look good. --remote makes sense because we've used it in the previous code snippet, but --image isn't. So let's just a dropdown below --remote dropdown.